### ISSUE TITLE
Change 9555387f697fd5507aae3f660c291c881b6f9e7e seems cause exports structure changes

### ISSUE THREAD

**Issue by whitetrefoil** [browserslist/caniuse-lite#71]
In [commit 9555387f697fd5507aae3f660c291c881b6f9e7e](https://github.com/browserslist/caniuse-lite/commit/19cfa6dc33844a8405ac61c639a6ef0af855cc9d#diff-ead8c9b65eef508575ae7610bc9458a46bb459496482285ecc8ab60708ab5af9L1)

```diff
-export { agents } from './agents'
-export { default as feature } from './feature'
-export { features } from './features'
-export { default as region } from './region'
+module.exports.agents = require('./agents')
+module.exports.feature = require('./feature')
+module.exports.features = require('./features')
+module.exports.region = require('./region')
```

causes

```js
require('caniuse-lite').agents;
// => { ie: ..., edge: ..., }
// now became
// => { agents: { ie: ..., edge: ..., }}

require('caniuse-lite').features;
// => { aac: ..., }
// now became
// => { features: { aac: ..., }}
```

This causes errors in using of autoprefixer, etc.

**Comment by whitetrefoil** [comment]
P.S. This should be the cause of #70

**Comment by ai** [comment]
Fixed fde289588

**Comment by ai** [comment]
The fix was released in 1.0.30001226
